MSAL/test/unit/native_auth/controllers/MSALNativeAuthSignInControllerTests.swift

```diff
@@ -393,7 +393,7 @@ final class MSALNativeAuthSignInControllerTests: MSALNativeAuthTestCase {
         helper.expectedCodeLength = expectedCodeLength
 
         let result = await sut.signIn(params: MSALNativeAuthSignInWithPasswordParameters(username: expectedUsername, password: expectedPassword, context: expectedContext, scopes: nil))
-        result.telemetryUpdate?(.failure(.init(message: "error")))
+        result.telemetryUpdate?(.failure(.init(identifier: 1, message: "error")))
```

Comment:
Should we use `SignInPasswordStartErrorType.browserRequired.rawValue` here to clarify what the identifier is? Same for the rest. Should we assign values like 0/1/2 to enums? [Apparently](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/enum) it should start with 0

Reply:
Ok, done. This error is just a stub and the number doesn't matter here. But I agree that using the `rawValue` of the enum type is clearer.